.github/workflows/cpath-pr-shadow.yml

```diff
@@ -0,0 +1,53 @@
+# This is a 'shadow' pipeline designed purely to allow status checks to be run on PRs that use path filtering
+name: Pathology PR Shadow
+on:
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - "hi-ml-cpath/**"
+      - ".github/workflows/cpath-pr.yml"
+      # When the core hi-ml libraries change, test the histo test too, to find problems with backwards compatibility
+      - "hi-ml-azure/**"
+      - "hi-ml/**"
+
+jobs:
+
+  cpath-flake8:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: 'echo "No build required" '
+
+  cpath-mypy:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: 'echo "No build required" '
+
+  cpath-pytest:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: 'echo "No build required" '
+
+  smoke_test_slidespandaimagenetmil:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: 'echo "No build required" '
+
+  smoke_test_tilespandaimagenetmil:
```

Comment:
is there a reason why we only selected these smoke tests as part of the shadow workflow? I believe this PR has been open for a while, we added other smoke tests recently, do we need smoke tests in the shadow workflow anyways?

Reply:
Also may I suggest renaming smoke tests? can we come up with a standard naming pattern for all? some have aml at the end, some have hyphens in between words some don't... we haven't agreed on a naming pattern before, would be great to add it in the docs somewhere to follow the same for future smoke tests.

Reply:
@kenza-bouzid As per Anton's latest comment these dependencies aren't needed anymore but yep I think it was just out of date with the latest smoke tests! 

Agreed on the naming for the smoke tests, I've opened [another issue](https://github.com/microsoft/hi-ml/issues/719) so this can be dealt with in a separate PR.